engine/src/main/java/org/terasology/engine/rendering/assets/skeletalmesh/SkeletalMeshDataBuilder.java

```diff
@@ -50,34 +50,6 @@ public SkeletalMeshDataBuilder addWeights(List<BoneWeight> weights) {
         return this;
     }
 
-    public SkeletalMeshDataBuilder addMesh(Bone bone, MeshBuilder builder) {
-        return addMesh(bone, builder.getMeshData());
-    }
-
-    public SkeletalMeshDataBuilder addMesh(Bone bone, StandardMeshData data) {
```

Comment:
where did this go?

Reply:
we don't use it in any capacity It was just a way to add meshes to bones.  I guess I would want to redo this as a skinned mesh so we're not doing this all on the CPU.